### PR TITLE
Added Travis testing w/ build changes, and fixed CPS subscription

### PR DESCRIPTION
**Pull request by jrheizelman**
This doesn't have any actual tests included yet, just makes sure the build works for now. In the future, I'll add some as I start writing.

### CODE REVIEW

**Comment by jrheizelman** [comment]
Finished working on CPS subscription to get it working even when trying to reuse a subscription name that already exists for the topic. In this case, originally I tried to get the old subscription and pull messages down, but that led to incorrect results due to old messages that hadn't been pulled down yet to come first, making the latency numbers potentially days. To fix, if the framework tries to create a subscription that already exists, it will delete the existing one and create a new one to be sure to only get messages sent after the load test started. 

I'll also add a something describing this to the readme, but I didn't think deleting the straggling old messages before running a load test with thousands of new meaningless ones was a big problem.

File: load-test-framework/src/main/java/com/google/pubsub/flic/cps/CPSRoundRobinSubscriber.java [hunk 1]
```diff
@@ -61,4 +62,11 @@ public CPSRoundRobinSubscriber(int numClients) throws Exception {
   public ListenableFuture<Subscription> createSubscription(Subscription request) {
     return pullClients.get(0).createSubscription(request);
   }
+  
+  /** Delete a Subscription. */
+  public ListenableFuture<Empty> deleteSubscription(Subscription request) {
+    DeleteSubscriptionRequest.Builder deleteBuilder = DeleteSubscriptionRequest.newBuilder();
+    deleteBuilder.setSubscription(request.getName());
+    return pullClients.get(0).deleteSubscription(deleteBuilder.build());
```
Comment (mdietz94) [inline load-test-framework/src/main/java/com/google/pubsub/flic/cps/CPSRoundRobinSubscriber.java]: Nit: I think the following is more idiomatic
```
return pullClients.get(0).deleteSubscription(
  DeleteSubscriptionRequest.newBuilder()
    .setSubscription(request.getName())
    .build());
```

File: load-test-framework/src/main/java/com/google/pubsub/flic/cps/CPSSubscribingTask.java [hunk 2]
```diff
@@ -70,9 +74,25 @@ public void execute() throws Exception {
     AcknowledgeRequest.Builder ackBuilder = AcknowledgeRequest.newBuilder();
     // Create a subscription for each topic.
     for (String topic : args.getTopics()) {
-      Subscription request =
-          subBuilder.setTopic(Utils.getCPSTopic(topic, args.getCPSProject())).build();
-      Subscription response = subscriber.createSubscription(request).get();
+      String[] splitOnColon = topic.split(":");
+      if(splitOnColon.length != 2)
+        throw new ParameterException("Parameter topics should be formatted 'topic:subscription'");
+      subBuilder.setName(Utils.getCPSSubscription(splitOnColon[1], args.getCPSProject()));
```
Comment (mdietz94) [inline load-test-framework/src/main/java/com/google/pubsub/flic/cps/CPSSubscribingTask.java]: This should be (builder functions return a builder):
```
subBuilder.setName(...)
          .setTopic(...);
```

File: load-test-framework/src/main/java/com/google/pubsub/flic/cps/CPSRoundRobinSubscriber.java (hunk 1 above)
Comment (jrheizelman) [inline load-test-framework/src/main/java/com/google/pubsub/flic/cps/CPSRoundRobinSubscriber.java]: Changed this in more recent commit

File: load-test-framework/src/main/java/com/google/pubsub/flic/cps/CPSSubscribingTask.java (hunk 2 above)
Comment (jrheizelman) [inline load-test-framework/src/main/java/com/google/pubsub/flic/cps/CPSSubscribingTask.java]: Changed this in more recent commit

**Comment by jrheizelman** [comment]
I made the builder changes you suggested in the recent commit to match the convention better.

**Pull request merged**